applications/models.py

```diff
@@ -299,6 +299,25 @@ def refund(self):
             change_status=ENROLL_CHANGE_STATUS_REFUNDED,
         )
 
+    @property
+    def is_eligible_to_skip_steps(self):
```

Comment:
I think we can just get rid of this and use `check_eligibility_to_skip_steps` directly in `get_or_create_bootcamp_application`

Reply:
In my opinion, we need to keep it as we need some checks to stop populating the jobma interview.

Reply:
Let my know what's your final opinion on that.